src/pretix/control/templates/pretixcontrol/orders/overview.html

```diff
@@ -17,63 +17,78 @@
+                            <th>{{ tup.0.num_expired|togglesum }}</th>
+                            <th>{{ tup.0.num_pending|togglesum }}</th>
                             <th>{{ tup.0.num_paid|togglesum }}</th>
+                            <th>{{ tup.0.num_total|togglesum }}</th>
                         </tr>
                     {% endif %}
                     {% for item in tup.1 %}
                         <tr class="item {% if tup.0 %}categorized{% endif %}">
                             <td>{{ item.name }}</td>
                             <td>
-                                <a href="{{ listurl }}?item={{ item.id }}&amp;provider={{ item.provider }}">
-                                    {{ item.num_total|togglesum }}
+                                <a href="{{ listurl }}?item={{ item.id }}&amp;status=c&amp;provider={{ item.provider }}">
+                                    {{ item.num_canceled|togglesum }}
                                 </a>
                             </td>
                             <td>
-                                <a href="{{ listurl }}?item={{ item.id }}&amp;status=ne&amp;provider={{ item.provider }}">
-                                    {{ item.num_pending|togglesum }}
+                                <a href="{{ listurl }}?item={{ item.id }}&amp;status=r&amp;provider={{ item.provider }}">
+                                    {{ item.num_refunded|togglesum }}
                                 </a>
                             </td>
                             <td>
-                                <a href="{{ listurl }}?item={{ item.id }}&amp;status=c&amp;provider={{ item.provider }}">
-                                    {{ item.num_canceled|togglesum }}
+                                <a href="{{ listurl }}?item={{ item.id }}&amp;status=e&amp;provider={{ item.provider }}">
+                                    {{ item.num_expired|togglesum }}
                                 </a>
                             </td>
                             <td>
-                                <a href="{{ listurl }}?item={{ item.id }}&amp;status=r&amp;provider={{ item.provider }}">
-                                    {{ item.num_refunded|togglesum }}
+                                <a href="{{ listurl }}?item={{ item.id }}&amp;status=n&amp;provider={{ item.provider }}">
+                                    {{ item.num_pending|togglesum }}
                                 </a>
                             </td>
                             <td>
                                 <a href="{{ listurl }}?item={{ item.id }}&amp;status=p&amp;provider={{ item.provider }}">
                                     {{ item.num_paid|togglesum }}
                                 </a>
                             </td>
+                            <td>
+                                <a href="{{ listurl }}?item={{ item.id }}&amp;provider={{ item.provider }}">
```

Comment:
Shouldn't this now link to something like ``status=np`` that we would first need to create?